src/components/PeerGroupsUpdate.tsx

```diff
@@ -0,0 +1,176 @@
+import React, {useEffect, useRef, useState} from 'react';
```

Comment:
Codacy has a fix for the issue: [Replace `useEffect,·useRef,·useState}·from·'react'` with `·useEffect,·useRef,·useState·}·from·"react"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
import React, { useEffect, useRef, useState } from "react";
```